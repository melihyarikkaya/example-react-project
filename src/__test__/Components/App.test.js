import React from 'react';
import { shallow } from 'enzyme';

import App from '../../components/App';

describe('App Component', () => {
  describe('Snapshot', () => {
    it('should match last snapshot', () => {
      const wrapper = shallow(<App />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
