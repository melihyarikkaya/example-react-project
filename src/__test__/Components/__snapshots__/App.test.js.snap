// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Snapshot should match last snapshot 1`] = `<AppRouter />`;
